WordPressKit/TransactionsServiceRemote.swift

```diff
@@ -151,12 +151,12 @@ public enum TransactionsServiceProduct {
 
 public struct CartResponse {
     let blogID: Int
-    let cartKey: Int
+    let cartKey: Any
```

Comment:
❓ I'm not very familiar with the `me/shopping-card` endpoint, so I'm not sure what to expect from this property of type `Any`. What are the possible values of this property?

Reply:
Normally the cartKey is an integer, but in the case of a no-site cart, the cart Key is a string with the value "no-site". Hence the change to `Any`.

Reply:
> Maybe a short documentation can be useful here.

Agreed! Added in 00c748e20adea36dd6208efcde49b4d336c61fae